forest/cli/src/cli/send_cmd.rs

```diff
@@ -5,121 +5,13 @@ use std::str::FromStr;
 
 use forest_json::message::json::MessageJson;
 use forest_rpc_client::{mpool_push_message, wallet_default_address};
+use forest_shim::econ::TokenAmount;
 use fvm_ipld_encoding::Cbor;
-use fvm_shared::{address::Address, econ::TokenAmount, message::Message, METHOD_SEND};
-use num::BigInt;
-use rust_decimal::prelude::*;
-use rust_decimal_macros::dec;
+use fvm_shared::{address::Address, message::Message, METHOD_SEND};
+use num::Zero as _;
```

Comment:
Why use `as _` here?

Reply:
Its a convention for using trait methods when the trait itself is never referred to (e.g in a bound)